### ISSUE TITLE
upload-sourcemaps not compatible with node < 14

### ISSUE THREAD

**Issue by awinograd** [expo/sentry-expo#177]
### Summary

I get the following error trying to require the upload-sourcemaps hook (i added a try catch in sentry-expo/upload-sourcemaps):

```
/Users/awinograd/programming/aviva/node_modules/sentry-expo/build/hooks/upload-sourcemaps.js:39
      SENTRY_ORG: config?.organization || process.env.SENTRY_ORG,
                         ^

SyntaxError: Unexpected token '.'
    at Module._compile (internal/modules/cjs/loader.js:895:18)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/Users/awinograd/programming/aviva/node_modules/sentry-expo/upload-sourcemaps.js:6:20)
    at Module._compile (internal/modules/cjs/loader.js:959:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
Unable to load postPublish hook: 'sentry-expo/upload-sourcemaps'. The module does not export a function.
Please fix your postPublish hook configuration
```

From what I can tell, optional chaining is not supported in node < 14. I'm currently on node 12.x LTS

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

bare

### What platform(s) does this occur on?

Android, iOS

### SDK Version (managed workflow only)

40

### Environment

```

  Expo CLI 4.2.1 environment info:
    System:
      OS: macOS 10.15.7
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 12.14.0 - ~/.nodenv/versions/12.14.0/bin/node
      Yarn: 1.22.10 - ~/.nodenv/versions/12.14.0/bin/yarn
      npm: 6.13.4 - ~/.nodenv/versions/12.14.0/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    Managers:
      CocoaPods: 1.10.0 - /Users/awinograd/.rbenv/shims/pod
    SDKs:
      iOS SDK:
        Platforms: iOS 14.3, DriverKit 20.2, macOS 11.1, tvOS 14.3, watchOS 7.2
      Android SDK:
        API Levels: 29, 30
        Build Tools: 28.0.3, 29.0.2, 30.0.2
        System Images: android-30 | Google APIs Intel x86 Atom
    IDEs:
      Android Studio: 4.1 AI-201.8743.12.41.6858069
      Xcode: 12.3/12C33 - /usr/bin/xcodebuild
    npmPackages:
      expo: ^40.0.0 => 40.0.1 
      react-native: 0.63.4 => 0.63.4 
    Expo Workflow: bare
```

### Reproducible demo or steps to reproduce from a blank project

run expo-publish using node 12

Semi related... I have a custom webpack config and had to add `sentry-expo` to  `dangerouslyAddModulePathsToTranspile` because of the null-coalescing operator. Not sure if the library bundling is just using the same thing an needs to change the target / add a babel transform

```
    await expoConfig(
      {
        ...env,
        offline: false,
        pwa: false,
        babel: {
          dangerouslyAddModulePathsToTranspile: ['@sentry/react-native', 'sentry-expo'],
        },
      },
      {
        ...argv,
      },
    ),
```

**Comment by cruzach** [comment]
gotchya, sorry about this- will remove optional chaining and null-coalescing, thanks very much @awinograd

**Comment by awinograd** [comment]
Not a problem! Thanks @cruzach. It was good motivation to upgrade to node 14 as it was about time anyway :)

**Comment by cruzach** [comment]
fixed in https://github.com/expo/sentry-expo/commit/329888ed38d5a5d72e0f2bd432f31e661a7ab5f0

**Comment by cruzach** [comment]
released in 3.1.2 (install with `yarn add sentry-expo@next`)
